firmware/boron-ins-fsm/src/stateMachine.cpp

```diff
@@ -126,10 +144,10 @@ void state0_idle() {
 
     Log.info("You are in state 0, idle: Door status, iAverage = 0x%02X, %f", checkDoor.doorStatus, checkINS.iAverage);
     // default timer to 0 when state doesn't have a timer
-    publishDebugMessage(0, checkDoor.doorStatus, checkINS.iAverage, 0);
+    publishDebugMessage(0, checkDoor.doorStatus, checkINS.iAverage, (millis() - timeSinceDoorClosed));
```

Comment:
Thanks for updating the last argument to this function. I think that will be helpful for debugging.